### PR TITLE
Improve README.md Windows

### PR DESCRIPTION
**Pull request by wagrenier**
I added some information on how to compile with VS2019 and add the Play! debugger to the binaries. Included the fixes that I did for some problems I had like the plugins folder not being found.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,12 +26,25 @@ To build for Windows you will need to have CMake installed on your system.
  ```
  ```
  # Not specifying -G will automatically generate 32-bit projects.
+ # This will generate a 64bits build with VS2017 compiler
  cmake .. -G "Visual Studio 15 2017 Win64" -DCMAKE_PREFIX_PATH="C:\Qt\5.10.1\msvc2017_64" -DUSE_QT=YES
+
+ # This will generate a 64bits build with VS2019 compiler
+  cmake .. -G "Visual Studio 16 2019" -DCMAKE_PREFIX_PATH="C:\Qt\5.15.2\msvc2019_64" -DUSE_QT=YES
```
Comment (Zer0xFF) [inline README.md:33]: starting from VS 16 2019, you have to specify arch using `-A x64` or `-A Win32`, and i do believe it defualts to 32 bits if you dont specify

File: README.md [hunk 2]
```diff
@@ -26,12 +26,25 @@ To build for Windows you will need to have CMake installed on your system.
...
  ```
 You can now build the project by opening the generated Visual Studio Solution or continue through cmdline:
  ```cmd
  cmake --build . --config Release
  ```
+
+ To add the Play! debugger to the binaries, simply add the following command to cmake
+ ```cmd
+   # To add the debugger to the build
```
Comment (Zer0xFF) [inline README.md:42]: you should probably be more specific here, this should be used in build generation setup, not during the building step

File: README.md (hunk 1 above)
Comment (wagrenier) [reply README.md:33]: On my side it seems to be compiling in `x64`, perhaps because I specified the `msvc2019_64` folder.

File: README.md (hunk 2 above)
Comment (wagrenier) [reply README.md:42]: I'll change the section it is in. I'll make a `Debugger` section and put it there.

**Pull request closed without merging**